NOTICE

```diff
@@ -44,6 +44,17 @@ The text of each license is the standard Apache 2.0 license.
 * ray (https://github.com/ray-project/ray)
     Files:
       ci/format.sh
+      bazel/fury_deps_setup.bzl
+      bazel/grpc-cython-copts.patch
+      bazel/grpc-python.patch
+
+* tensorflow (https://github.com/tensorflow/tensorflow)
+    Files:
+      bazel/arrow/pyarrow_configure.bzl
+
+* cython-abseil-sample (https://github.com/messense/cython-abseil-sample)
+    Files:
+      python/pyfury/includes/libabsl.pxd
```

Comment:
It should be under MIT license but this block is for Apache-2.0?

Reply:
fixed